tests/Common/DevBranchesTest.php

```diff
@@ -113,99 +150,120 @@ function ($branch) use ($branchName) {
         try {
             $branches->deleteBranch($branchId);
         } catch (ClientException $e) {
-            $this->assertSame(
-                sprintf('Branch not found'),
-                $e->getMessage()
-            );
+            $this->assertSame(404, $e->getCode());
+            $this->assertSame(sprintf('Branch not found'), $e->getMessage());
         }
 
         // now branch can be created with same name as deleted branch
-        $newBranch = $branches->createBranch($branchName);
+        $branches->createBranch($branchName);
     }
 
-    public function testOrgAdminCanDeleteBranchCreatedByAdmin()
+
+    public function testGuestRoleBranchesPermissions()
```

Comment:
Nemohl bys tenhle test nechat v testAdminRoleBranchesPermissions? Takhle zbytečně vytváříš a zahazuješ ty branche a testuješ ~ to samé...